app/helpers/facility_order_status_helper.rb

```diff
@@ -34,13 +34,6 @@ def new_or_in_process_orders
     end
   end
 
-  def problem_orders
```

Comment:
This might be a little oddly named since it's called `problem_orders` but it actually returns OrderDetails. not necessarily worth changing, but I noticed that since you moved it.